functions/packages.bash

```diff
@@ -258,11 +258,14 @@ mqtt_setup() {
   echo -n "$(timestamp) [openHABian] Beginning the MQTT broker Eclipse Mosquitto installation... "
   if [[ -n $INTERACTIVE ]]; then
     if (whiptail --title "MQTT installation?" --yes-button "Continue" --no-button "Cancel" --yesno "$introText" 14 80); then echo "OK"; else echo "CANCELED"; return 0; fi
+  else
```

Comment:
Is this build related ?
If no please separate PRs

Reply:
Yes, actually. For some stupid reason without that check it was failing every time. Beats me.